app/views/admin/products_v3/_delete_modals.html.haml

```diff
@@ -0,0 +1,17 @@
+-# object_type can be 'variant' or 'product'
+%div{ id: "#{object_type}-delete-action-modals" }
+  - base_translation_key = ".delete_#{object_type}_modal"
+  - object_ids.each do |object_id|
+    - delete_modal = ConfirmModalComponent.new(id: "delete_#{object_type}_#{object_id}",
```

Comment:
This is a lot of additional modals. I would prefer a more generic solution where we have one modal for all of the variants and products. The text is the same. It just needs to trigger a reflex defined on the button.

Reply:
It's been fixed here: https://github.com/openfoodfoundation/openfoodnetwork/pull/11846/commits/6ed447b6ac82f675f5424f45cea21b6d18cb93a8

- However, we need at least 2 modals. One for the product and other for the variant.
- Because they have slightly different text (_or translations_) for product and variant modal
- So the above fix creates two static modals for product and variant and its current id for the reflex action params is dynamically set by stimulus action.
- Please review and let me know what you think about it. Thanks.

Reply:
It's been fixed here: https://github.com/openfoodfoundation/openfoodnetwork/pull/11846/commits/6ed447b6ac82f675f5424f45cea21b6d18cb93a8
- However, we need at least 2 modals. One for the product and other for the variant.
- Because they have slightly different text (_or translations_) for product and variant modal
- So the above fix creates two static modals for product and variant and its current id for the reflex action params is dynamically set by stimulus action.
- Please review and let me know what you think about it. Thanks.

Reply:
Great solution 👍